site/src/router.tsx

```diff
@@ -426,6 +426,23 @@ export const router = createBrowserRouter(
 						</Route>
 					</Route>
 
+					{/**
+					 * @todo 2024-10-24 - MES - The current deployment page is
+					 * almost empty, which isn't great for two reasons:
+					 *
+					 * 1. Even though none of our links lead to the page, the
+					 *    user is still able to navigate straight there by
+					 *    accident.
+					 * 2. If the user has access to some deployment pages, but
+					 *    tries accessing one that they shouldn't be able to
+					 *    access, we have to reroute them somewhere. The base
+					 *    deployment route is the only safe option, because not
+					 *    even the general page is accessible by everyone.
+					 *
+					 * Should update the base /deployment page so that there's
+					 * something there, but that will require coordination with
+					 * the design team.
+					 */}
```

Comment:
I really wish I could've added some content here, but it felt really out of scope. At the very least, better a janky experience over a broken one

Reply:
Maybe something to think together with @chrifro